.github/workflows/test-all.yml

```diff
@@ -272,11 +279,39 @@ jobs:
           python -m pip install -r tools/requirements.txt
 
       - name: "Generate test-status.json"
-        id: report
-        run: python tools/generate_test_status.py ${{ github.repository }} "$GITHUB_RUN_ID" "$GITHUB_SHA" $(git rev-parse --short "$GITHUB_SHA")
-
-      - name: "Output test-status"
         id: test-status
         run: |
-          TEST_STATUS=$(cat test-status.json)
-          echo "::set-output name=test-status::$TEST_STATUS"
+          ROOT='data/reports'
+          DIR_REL=$(python tools/generate_test_status.py ${{ github.repository }} "$GITHUB_RUN_ID" "$GITHUB_SHA" ${{ github.event.inputs.which-gap }})
```

Comment:
What kind of value will `DIR_REL` end up being here? This kind of info often is helpful to have in a comment (e.g. for debugging purposes), and saves one from having to next analyze that python script. (And yes, my code is often lacking these, too -- because it's easy to forget and one doesn't realize it as a problem while writing the code, as then of course one has all this info in one's head... as such, I appreciate such points for any of my code, too!)